### DISCUSSION TITLE
Debug error

### DISCUSSION THREAD

**Discussion by nikkolasg** [protobuf-c/protobuf-c#643] [Q&A]
Hi, 

I'm using your library, and I've got a problem when unpacking a message. I'd like to see any error that might help me pinpoint the problem. I've found `PROTOBUF_C_UNPACK_ERROR` somewhere in the code but I can't find any way to use it properly. 
Would you mind sharing a snippet to show how can we retrieve the error messages that your library logs ? 
I've already tried 
```
#define PROTOBUF_C_UNPACK_ERROR(args...) printf(args);
```
or just add `-DPROTOBUF_C_UNPACK_ERROR` to the compilation flags.

But I see nothing. I'm not a C expert, so please forgive me if that sounds a stupid question ... :)

Thanks

**Comment by lipnitsk** [comment]
This library is pretty stable at this point. Likely something is wrong with your message or the IDL specifying the message. Please reopen and provide the sample message bytestream and a proto file if the issue persists.
